sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -285,9 +285,9 @@ private void onBatchExport(final List<SpanData> spans) {
                 public void run() {
                   // In case of any exception thrown by the service handlers catch and log.
                   try {
-                    spanExporter.export(spans);
-                  } catch (Throwable t) {
-                    logger.log(Level.WARNING, "Exception thrown by the export.", t);
+                    spanExporter.export(spans).get();
```

Comment:
this is another point at which we should probably have a follow-on issue, so we don't lose track of the fact that we're making something sync that could be async.

Reply:
I will be applying the CompletableMetricStatus here too.